Add CSS for blockquote for warnings

mabelzhang opened the pull request:
@azeey

CSS for blockquotes to show up as a yellow warning banner.

Whether it works on the actual website, we'll have to see...

I tested using the ngx-markdown [preview editor](https://stackblitz.com/edit/ngx-markdown) linked from the [official package](https://www.npmjs.com/package/ngx-markdown):

In app.component.ts, there’s already a line for blockquote:
```
> Blockquote to the max`;
```

Add a line before the tick `:
```
<blockquote class="warning">Second blockquote</blockquote>
```

In styles.css, add:
```
blockquote.warning {
  border-left: 3px solid rgb(228, 167, 2);
  padding-left: 12px;
  color: rgb(138, 108, 64);
  background: rgb(252, 248, 228);
}
```

The Second blockquote in the preview panel should change to yellow.

mabelzhang committed:
add CSS for blockquote for warnings

Signed-off-by: Mabel Zhang <mabel@openrobotics.org>

azeey commented on src/styles.scss:
```diff
@@ -164,6 +164,13 @@ footer.docs {
   width: calc(100% - 280px);
 }
 
+blockquote.warning {
```
I would suggest we apply this to any tag with a warning class.

```suggestion
.warning {
```

mabelzhang replied:
4d5ffcf

azeey commented on src/styles.scss:
```diff
@@ -164,6 +164,13 @@ footer.docs {
 
+blockquote.warning {
+  border-left: 3px solid rgb(228, 167, 2);
+  padding-left: 12px;
```
This is currently how it looks:
![image](https://user-images.githubusercontent.com/206116/203646346-cbb78348-13ca-4512-820d-7d6944290cbf.png)

Changing to
```suggestion
  padding: 15px;
```
yields

![image](https://user-images.githubusercontent.com/206116/203646523-cceafb48-f3fa-4700-b674-b5eba6dd6e63.png)


The left indentation is because it's a `blockquote`. I suggest we change it to `<div>` and add a vertical margin

```
margin: 15px 0;
```
to yield

![image](https://user-images.githubusercontent.com/206116/203646776-e8ad6571-a0c9-4ff5-ae3b-062d21943b9f.png)

mabelzhang replied:
Nice! Thanks for playing with the numbers! [4d5ffcf](https://github.com/gazebo-web/gazebosim-web-frontend/pull/12/commits/4d5ffcf6ee958f21915690a87c81895f8df8a31c)

azeey reviewed (commented):
Thanks @mabelzhang! I made a couple of suggestions regarding spacing after trying it out. Otherwise, looks great!

mabelzhang committed:
styling from PR comments

Signed-off-by: Mabel Zhang <mabel@openrobotics.org>

mabelzhang commented:
Addressed comments.

How were you able to pull the docs repo into the Docker images? I was able to run the Docker images, but my Docs tab just shows a blank page. So I just pasted the HTML on another page to test it :sweat_smile: Do I need to run something from the docs repo or put the docs repo in a specific location?

azeey commented:
Yeah, I believe you need to clone https://github.com/gazebosim/docs/ into the `docs` directory inside the backend source code.

mabelzhang commented:
:thinking: Copying to backend root directory didn't work for me. I tried copying into frontend `src/app`, which seems a reasonable place as it also has `showcase` and `features`, which are other tabs. But that didn't work either. Oh well, we don't need that to work for this.
